### ISSUE TITLE
Mint-X theme using wrong widgets

### ISSUE THREAD

**Issue by mfreeman72** [linuxmint/mint-themes#397]
On most standard window title bars (a Nemo window, for example), Mint-X widgets are no longer their usual theme. They are much thinner and lighter, and don't change at all when you raise or lower a window behind or above another. Because of this, they don't look good and are more difficult to see, and you can't tell when a window is active or in the background. However, when using a window that has that GTK-3/4-style big slab window title bar that Gnome apps tend to use, the widgets are back to normal.

Examples - 

Mint-X widgets on window with standard window title bar:
![Screenshot from 2022-08-15 15-10-59](https://user-images.githubusercontent.com/7842664/184728035-bff5cf54-ee0a-4685-a66a-afca9dc08e0c.png)

Mint-X widgets on window with Gnome-3/4-style window bar:
![Screenshot from 2022-08-15 15-10-19](https://user-images.githubusercontent.com/7842664/184728203-14ae18df-6d09-436e-b70d-5de2079b2d89.png)

No other themes seem to have this problem. Only Mint-X.

**Comment by SebastJava** [comment]
Same issue here: https://github.com/linuxmint/muffin/issues/629
I just didn't know where to open that issue, what was causing this.

**Comment by mfreeman72** [comment]
Well, someone over on the Linux Mint blog figured out what's going on, I think. The widgets for the normal windows (my top image above) are not always controlled by the window theme, but instead by the *icon* theme. So if I use the Mint-X window theme and the Mint-Y icon theme (which is my personal preference), I will get these conflicting results that you see in the screenshots above. But if I match up the themes so that I'm using Mint-X for both the window theme and the icon theme, it follows the expected theme in all windows. This is the case with both Mint-X and Mint-Y themes. However, there are some themes where this isn't the case. For example, Adapta window widgets *always* follow the expected Adapta theme, no matter what I have the icon theme set as. This is really weird and massively confusing and inconsistent. Why are this one piece of the window theme suddenly tied to the icon theme, of all things?! It makes absolutely no sense.

**Comment by JosephMcc** [comment]
Yes, this is an issue with the theme. It just hasn't been changed yet. I haven't decided exactly how I want to go about fixing it yet.

**Comment by mfreeman72** [comment]
@JosephMcc Ok. Thank you for the information!

**Comment by mfreeman72** [comment]
@JosephMcc Any updates on the status of this? This issue makes it difficult to use a different icon theme than the one matching your window theme. Changing the icon theme makes your window widgets look really out of place, which (in my opinion) it shouldn't do. For example, I only use Mint-X, because I detest flat themes for windows borders and buttons. But I prefer the more modern look of the Mint-Y icon theme for everything else. Selecting this combination causes window widgets to look inconsistent with the window theming. Even worse, the window widgets on client-side decorations follow the window theme (the way I want it), but server-side decorations follow the icon theme. This makes theming wildly inconsistent unless you match the icon theme to the window theme in all cases, which makes the ability to change icon themes in the theme settings window pointless.

Can window themes be made to check for matching icon themes and use widgets from that icon theme instead of the icon theme selected for the rest of the system icons, and only falling back to those if no matching icon themes are found? That way everything is consistent across all windows and themes, as much as possible.

**Comment by JosephMcc** [comment]
I have a half done PR to fix this. I just need to get around to finishing it. It's just some tedious work and I've been really busy. This is 100% percent a theme issue and nothing else really needs to be done to deal with this.

**Comment by mfreeman72** [comment]
Thanks for the update! I appreciate you taking time to respond
